src/Compilers/VisualBasic/Portable/Symbols/MethodSymbol.vb

```diff
@@ -100,6 +100,25 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
+
+        ''' <summary>
+        ''' Always returns false because the 'readonly members' feature is not available in VB.
+        ''' </summary>
+        Public ReadOnly Property IsDeclaredReadOnly As Boolean Implements IMethodSymbol.IsDeclaredReadOnly
```

Comment:
Consider making these private explicit implementations so that the APIs are not accessible from `MethodSymbol`.

Reply:
Makes sense. Looks like that will be consistent with how `TypeSymbol.IsReadOnly` works. https://github.com/dotnet/roslyn/pull/33528/files#diff-06e6f52c4f92becc55b74f5454eccf60R575